cypress/integration/landing.js

```diff
@@ -51,10 +64,13 @@ describe('Landing page', () => {
   });
 
   it('Has 10 certifications', function() {
```

Comment:
Should we rename this test?

Reply:
```suggestion
  it('Has links to all the certifications', function() {
```
Is that what you had in mind?  Feel free to add any improvements.